### ISSUE TITLE
508-defect-2 [COGNITION]: BAM2 Debt Letters - Debt Detail notice and balance information must have a programmatic relationship

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16786]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a screen reader user, I need to understand the relationship between my debt notice, starting amount, and current amount data. Right now the labels are read, then the values instead of a label then its related value.

## Details

The introductory data for the following items need to be coded as a list, possibly a description list:

* Date of first notice
* Original debt amount
* Current balance

Screenshot and proposed code snippet below.

## Acceptance Criteria

- [ ] Data is presented correctly to screen readers and assistive technology. This can be tested by confirming users hear text like "Date of first notice, colon, December 19, 2014" or "Original debt amount, colon, dollar sign one-hundred dot zero zero".
- [ ] No axe browser violations reported
- [ ] HTML for description list is well formed and validates
- [ ] No visual change to the layout

## Environment

* https://staging.va.gov/manage-va-debt/your-debt/debt-detail
* Test user 1

## Solution (if known)

```html
<div class="vads-u-display--flex vads-u-flex-direction--row">
  <dl class="vads-u-display--flex vads-u-flex-direction--column vads-u-margin-right--2">
    <dt class="vads-u-margin-y--0 vads-u-font-weight--bold">
      Date of first notice:
    </dt>
	<dd class="vads-u-margin-y--0">
      December 19, 2014
    </dd>
    <dt class="vads-u-margin-y--1 vads-u-font-weight--bold">
      Original debt amount:
    </dt>
	<dd class="vads-u-margin-y--0">
      $100.00
    </dd>
    <dt class="vads-u-margin-y--0 vads-u-font-weight--bold">
      Current balance:
    </dt>
	<dd class="vads-u-margin-y--0">
      $80.00
    </dd>
  </dl>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

![Screen Shot 2020-12-01 at 4 35 07 PM](https://user-images.githubusercontent.com/934879/100805430-43f37e80-33f4-11eb-99f7-804ca3317f24.png)

**Comment by bradl3yC** [comment]
@1Copenut Ready for validation

**Comment by 1Copenut** [comment]
@bradl3yC Can we make the `<dd>` elements left-aligned to match the original design? Otherwise this is good to go. Spot checked it with the big three screen readers as part of another item.

**Comment by bradl3yC** [comment]
@1Copenut this has been resolved - should be in staging in the next 30 mins to 1 hour

**Comment by 1Copenut** [comment]
@bradl3yC This does get us what we're looking for in the semantic relationship, which captures the ticket scope. If there's time to tweak the CSS, it would be worth making the `<dl>` 100% parent width, setting a fixed width on the `<dt>` and removing the space-between declaration. This will give us a cleaner layout on mobile. LMK if you'd like that as a separate issue ticket.

---
![Screen Shot 2020-12-07 at 4 52 17 PM](https://user-images.githubusercontent.com/934879/101415102-f0c67380-38ac-11eb-9577-1a0ed3b20b1e.png)

**Comment by bradl3yC** [comment]
@1Copenut made the changes you suggested - should be in staging now

**Comment by 1Copenut** [comment]
Looking good! Moving this one to close @bradl3yC
